src/test/emulators/storage/rules/manager.spec.ts

```diff
@@ -1,93 +1,104 @@
+import * as fs from "fs";
+import * as path from "path";
 import { expect } from "chai";
 import { tmpdir } from "os";
-import { v4 as uuidv4 } from "uuid";
 
-import { FirebaseError } from "../../../../error";
-import { StorageRulesFiles, TIMEOUT_MED } from "../../fixtures";
-import { StorageRulesManager } from "../../../../emulator/storage/rules/manager";
+import { StorageRulesFiles, TIMEOUT_LONG } from "../../fixtures";
+import {
+  createStorageRulesManager,
+  StorageRulesManager,
+} from "../../../../emulator/storage/rules/manager";
 import { StorageRulesRuntime } from "../../../../emulator/storage/rules/runtime";
 import { Persistence } from "../../../../emulator/storage/persistence";
-import { RulesetOperationMethod } from "../../../../emulator/storage/rules/types";
+import { RulesetOperationMethod, SourceFile } from "../../../../emulator/storage/rules/types";
+import { readFile } from "../../../../fsutils";
+import { isPermitted } from "../../../../emulator/storage/rules/utils";
 
 // TODO(hsinpei: Make this an integration test
 describe.skip("Storage Rules Manager", function () {
   const rulesRuntime = new StorageRulesRuntime();
-  const rulesManager = new StorageRulesManager(rulesRuntime);
+  const rules = [
+    { resource: "bucket_0", rules: StorageRulesFiles.readWriteIfTrue },
+    { resource: "bucket_1", rules: StorageRulesFiles.readWriteIfAuth },
+  ];
+  const opts = { method: RulesetOperationMethod.GET, file: {}, path: "/b/bucket_2/o/" };
+  let rulesManager: StorageRulesManager;
 
   // eslint-disable-next-line @typescript-eslint/no-invalid-this
-  this.timeout(TIMEOUT_MED);
+  this.timeout(TIMEOUT_LONG);
```

Comment:
do we still need this now that this is an integration test?

Reply:
Yeah, I do see some [examples](https://github.com/firebase/firebase-tools/blob/c8337ee60e82a54ce877e71e7e1e342906df5ffb/scripts/extensions-emulator-tests/tests.ts#L32) of this being used in other integration tests.